Objects/typeobject.c

```diff
@@ -3700,11 +3701,17 @@ object_new(PyTypeObject *type, PyObject *args, PyObject *kwds)
         joined = PyUnicode_Join(comma, sorted_methods);
         if (joined == NULL)
             goto error;
+        method_count = PyObject_Length(abstract_methods);
+        if (method_count == -1)
+        {
+            goto error;
+        }
```

Comment:
To be consistent with the rest of this function, there should be no braces here.

Reply:
Done.